### PR TITLE
Updating to use Dapper and fix for capability selector

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/Hooks.cs [hunk 1]
```diff
@@ -20,10 +20,11 @@ public void AfterScenario()
...
 
-            if (_test.solution != null)
-            {
-                SqlHelper.DeleteSolution(_test.solution.Id, _test.connectionString);
-            }
+            //if (_test.solution != null)
+            //{
+            //    _test.solution.Delete(_test.connectionString);
+            //}
+
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/Hooks.cs]: why aren't you deleting the solution?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature [hunk 2]
```diff
@@ -12,30 +12,19 @@ Scenario: Solution Details
 	And Solution Full Description
 	And About Solution URL
 	And Contact Details
-	And Last Updated Date
 	And list of Capabilities
 	And Solution ID
 
-Scenario: Attachment Download Link Displayed
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature:19]: where is this covered if you're removing it?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature [hunk 3]
```diff
@@ -12,30 +12,19 @@ Scenario: Solution Details
...
 	When the User is viewing the Solution Page
 	Then the page will contain an indication that the Solution meets the criteria for a Foundation Solution Set
 
 Scenario: Capabilities displayed correctly
-	Given that a User views a Solution
+	Given that a User views a Solution	
 	When the User is viewing the Solution Page
 	Then the capabilities listed match the expected capabilities in the database
 
-Scenario: Download more information
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature:34]: where is this covered if you're removing it?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/NoSelectionCriteriaBrowse.feature [hunk 4]
```diff
@@ -1,16 +0,0 @@
-﻿Feature: No-selection criteria browse
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/NoSelectionCriteriaBrowse.feature:1]: where is this covered if you're removing it?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/FoundationSolutionCapabilityFilteredBrowse.feature [hunk 5]
```diff
@@ -1,11 +0,0 @@
-﻿Feature: Foundation Solution (Single) Capability-filtered browse
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/FoundationSolutionCapabilityFilteredBrowse.feature:1]: where is this covered if you're removing it?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/CapabilityFilteredBrowse.feature [hunk 6]
```diff
@@ -1,33 +0,0 @@
-﻿Feature: Capability-filtered browse
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/CapabilityFilteredBrowse.feature:1]: where is this covered if you're removing it?

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature (hunk 3 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature:34]: This was only ever a placeholder until more functionality was included. As it is now obsolete, I have removed it

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature (hunk 2 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature:19]: DAPI test task added to backlog

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/NoSelectionCriteriaBrowse.feature (hunk 4 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/NoSelectionCriteriaBrowse.feature:1]: All existing non-foundation tests cover this functionality

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/CapabilityFilteredBrowse.feature (hunk 6 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/CapabilityFilteredBrowse.feature:1]: Part of next story covers this

File: src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/FoundationSolutionCapabilityFilteredBrowse.feature (hunk 5 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/FoundationSolutionCapabilityFilteredBrowse.feature:1]: This was only ever a placeholder until more functionality was included. As it is now obsolete, I have removed it

**Review by talawson05 — APPROVED** [review]

**Pull request merged**